sql/qaqc/qaqc_field_distribution.sql

```diff
@@ -0,0 +1,23 @@
+DROP TABLE IF EXISTS field_dist_qaqc;
+CREATE TABLE IF NOT EXISTS field_dist_qaqc(
+    field_name character varying,
+    result character varying
+);
+
+INSERT INTO field_dist_qaqc (
+    SELECT 'Job_Type' as field_name,
+            jsonb_agg(json_build_object('job_type',tmp.job_type,'count',tmp.count)) as result
+    FROM(
+        SELECT job_type, COUNT(DISTINCT job_number) as count
+        FROM final_devdb
+        WHERE date_lastupdt >= CAST(:PRE_DATE AS VARCHAR)
```

Comment:
from the other [qaqc scripts](https://github.com/NYCPlanning/db-developments/search?p=1&q=date_lastupdt&type=code) used the date_lastupdt which tells me this should work. But I would like go through the logic a little carefully about this